### PR TITLE
Temp calibration

### PR DESCRIPTION
**Pull request by eucalvo**
fix issue #174

### CODE REVIEW

**Review by jgfoster — APPROVED** [review]
Note that this addresses two issues:
- Fix problem with temperature calibration (#174)
- Show "current" values for time when setting new values (#164)

File: src/Devices/TempProbe_TC.h [hunk 1]
```diff
@@ -91,6 +91,9 @@ class TempProbe_TC {
   static TempProbe_TC* instance();
   static void reset();
 
+  float getCorrection() {
+    return correction;
+  }
   uint16_t getResistance() {
```
Comment (jgfoster) [inline src/Devices/TempProbe_TC.h:97]: Replace this with `getUncorrectedRunningAverage()`.

File: src/UIState/TemperatureCalibration.cpp [hunk 2]
```diff
@@ -9,10 +9,10 @@
 
 void TemperatureCalibration::setValue(float value) {
   TempProbe_TC *pProbe = TempProbe_TC::instance();
-  float average = pProbe->getRunningAverage();
-  pProbe->setCorrection(value - average);
+  float uncorrectedAverage = pProbe->getRunningAverage() - pProbe->getCorrection();
```
Comment (jgfoster) [inline src/UIState/TemperatureCalibration.cpp:12]: Use `getUncorrectedRunningAverage()`.

**Review by jgfoster — CHANGES_REQUESTED** [review]
Per discussion with @prestoncarman

**Review by jgfoster — APPROVED** [review]

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**